src/components/library-item/library-item.jsx

```diff
@@ -70,13 +73,68 @@ class LibraryItem extends React.PureComponent {
                         src={this.props.iconURL}
                     />
                 </div>
+                {this.props.insetIconURL ? (
+                    <div className={styles.libraryItemInsetImageContainer}>
+                        <img
+                            className={styles.libraryItemInsetImage}
+                            src={this.props.insetIconURL}
+                        />
+                    </div>
+                ) : null}
                 <div
-                    className={styles.featuredText}
+                    className={this.props.extensionId ?
+                        classNames(styles.featuredExtensionText, styles.featuredText) : styles.featuredText
+                    }
                 >
                     <span className={styles.libraryItemName}>{this.props.name}</span>
                     <br />
                     <span className={styles.featuredDescription}>{this.props.description}</span>
                 </div>
+                {this.props.bluetoothRequired || this.props.internetConnectionRequired || this.props.collaborator ? (
+                    <div className={styles.featuredExtensionMetadata}>
+                        <div className={styles.featuredExtensionRequirement}>
+                            {this.props.bluetoothRequired || this.props.internetConnectionRequired ? (
+                                <div>
+                                    <div>
+                                        <FormattedMessage
+                                            defaultMessage="Requires"
+                                            description="Label for extension hardware requirements"
+                                            id="gui.extensionLibrary.requires"
+                                        />
+                                    </div>
+                                    <div
+                                        className={styles.featuredExtensionMetadataDetail}
+                                    >
+                                        <img
+                                            src={this.props.bluetoothRequired ?
+                                                bluetoothIconURL :
+                                                internetConnectionIconURL
+                                            }
+                                        />
+                                    </div>
+                                </div>
+                            ) : null}
+                        </div>
+                        <div className={styles.featuredExtensionCollaboration}>
+                            {this.props.collaborator ? (
+                                <div>
+                                    <div>
+                                        <FormattedMessage
```

Comment:
Why don't you use {} formatting and "values"?

Reply:
oooh, what do you mean by that?  jsx and React are new to me!  Thanks!

Reply:
It's called a _formatted_ message because you can substitute values in. You can then pass in all the named substitutions using a `values` prop.

For example, `"Hello, {name}"` and then a `values` prop of `{name: "Anna"}`

Reply:
Oh, whoa! Thanks so much, I will try it!

Reply:
Docs: https://github.com/yahoo/react-intl/wiki/Components#formattedmessage
Note: When passing an object literal as a prop, you need to double up the curly braces

Reply:
@evhan55 I don't think this would be appropriate here, since we (a) will not be translating the names of the collaborators and (b) are putting the collaborator name in a distinct place, we cannot have the contributor rearranged with the other text.

Reply:
@paulkaplan Actually this line here is for translating the label 'Collaboration with', I think that should be ok, right?  We will not be translating the collaborator names.

Reply:
@evhan55, you're right, your change is a good thing... it's just that the thing @apple502 and I mentioned (using curly braces and substitution) doesn't work here because we are putting the collaborator names in a separate tag.

Ideally, we'd use substitution, because in some languages the word order might change. With a long list like this though, I'm sure there's a way of phrasing it that would allow for this word order to be maintained

Reply:
@joker314 Oh, I'm learning all the things!  Thank you for explaining.